remoulade/middleware/retries.py

```diff
@@ -85,6 +80,16 @@ def after_process_message(self, broker, message, *, result=None, exception=None)
         min_backoff = self.get_option("min_backoff", broker=broker, message=message)
         max_backoff = self.get_option("max_backoff", broker=broker, message=message)
         max_backoff = min(max_backoff, DEFAULT_MAX_BACKOFF)
```

Comment:
I don't see any reason for that. I think the default should be lower, maybe 1 hour, and then ok to add a `MAX_BACKOFF` of one week but it is extreme anyway, I would just remove it